It seems CI broken in this PR. could you fix this?
```
Checking whether SageMath should install SPKG gfortran...
checking the version of gfortran... 15
real_configure: will use system package and not install SPKG gfortran
-----------------------------------------------------------------------------
Checking whether SageMath should install SPKG openblas...
checking for Accelerate/Accelerate.h... no
real_configure: error: Cannot locate Accelerate headers. Make sure XCode CL tools are installed.
```
seems it broke the sage-the-distro ci for all linux systems

--- a/configure.ac
+++ b/configure.ac
@@ -6,7 +6,7 @@
 #                     2014-2017 François Bissey
 #                     2016-2022 Matthias Koeppe
 #                     2017-2018 Erik M. Bray
-#                     2018      Dima Pasechnik
+#                     2018-2026 Dima Pasechnik
 #                     2020      Jonathan Kliem
 #                     2021-2023 Michael Orlitzky
 #
@@ -342,20 +342,21 @@ m4_pushdef([SAGE_BOOTSTRAP_URL],[See https://doc.sagemath.org/html/en/reference/
    ])
 
 # choose BLAS/LAPACK on Darwin
-AC_ARG_WITH([darwin-accelerate],
+SAGE_DARWIN_PKG_CONFIG_PATH=
+case $host in
+  *-*-darwin*)
+   AC_ARG_WITH([darwin-accelerate],
             [AS_HELP_STRING([--with-darwin-accelerate={yes (default)},no}],
                             [use Darwin (macOS) Accelerate BLAS/LAPACK])],
             [SAGE_DARWIN_ACCELERATE="$withval"],
             [SAGE_DARWIN_ACCELERATE="yes"])
-AC_SUBST([SAGE_DARWIN_ACCELERATE])
-
-SAGE_DARWIN_PKG_CONFIG_PATH=
-case $host in
-    *-*-darwin*)
+   AC_SUBST([SAGE_DARWIN_ACCELERATE])
    AS_IF([test x$SAGE_DARWIN_ACCELERATE = xyes],
     [SAGE_DARWIN_PKG_CONFIG_PATH=`pwd`"/build/platform/darwin/accelerate/pkgconfig:"])
     ;;
-    *)
+  *)
+   SAGE_DARWIN_ACCELERATE="no"
+   ;;
 esac
 
 # Check for zlib

